qutebrowser/browser/downloads.py

```diff
@@ -873,6 +926,12 @@ def remove_items(self, downloads):
                 download.deleteLater()
         self.endRemoveRows()
 
+    def update_indexes(self):
+        """Update indexes of all DownloadItems"""
+        for i, d in enumerate(self.downloads, 1):
+            if not d.index == i:
```

Comment:
I'm not sure if this is still needed (see comment below), but if so, it should be written as `if d.index != i:` instead.